modules/tinymce/src/core/test/ts/browser/textpatterns/TextPatternsForcedRootBlockFalseTest.ts

```diff
@@ -3,16 +3,15 @@ import { TinyAssertions, TinyHooks } from '@ephox/wrap-mcagar';
 
 import Editor from 'tinymce/core/api/Editor';
 import ListsPlugin from 'tinymce/plugins/lists/Plugin';
-import TextPatternPlugin from 'tinymce/plugins/textpattern/Plugin';
 
-import * as Utils from '../module/test/Utils';
+import * as Utils from '../../module/test/TextPatternsUtils';
 
-describe('browser.tinymce.plugins.textpattern.TextPatternPluginForcedRootBlockFalseTest', () => {
+describe('browser.tinymce.textpatterns.TextPatternsForcedRootBlockFalseTest', () => {
```

Comment:
```suggestion
describe('browser.tinymce.core.textpatterns.TextPatternsForcedRootBlockFalseTest', () => {
```
I must have missed this one previously sorry.